src/api/app/assets/javascripts/webui2/tabs.js

```diff
@@ -0,0 +1,43 @@
+$.fn.hasOverflow = function() {
```

Comment:
This would need to somehow calculate in the width of the dropdown....

![screenshot from 2018-07-30 16-54-38](https://user-images.githubusercontent.com/514785/43405066-4af945e8-9419-11e8-9ee3-9a3686278049.png)

Reply:
Please have another look.

Reply:
Awesome. Now all you have to do is to make it it [right aligned](https://getbootstrap.com/docs/4.1/components/dropdowns/#menu-alignment)

Reply:
Done